docs/getting-started.md

```diff
@@ -0,0 +1,80 @@
+| [PXE](../install/pxe-boot-install) | ISO, Linux kernel image (vmlinuz), initrd, SquashFS image | Directory on the HTTP server (for serving boot files); iPXE boot scripts (for automatic installation); DHCP server configuration |
+
+## 3. Prepare the cluster configuration requirements. 
+
+- Cluster token: ASCII string that nodes use when joining the cluster 
+- Fixed IP address for each node: May be assigned statically or using DHCP (host reservation) 
+- Fixed virtual IP address (VIP) to be used as the cluster management address: VIP that nodes point to when joining the cluster and that you connect to when performing administration tasks after the cluster is deployed 
+- Addresses of DNS servers, NTP servers, and the proxy server (if necessary) 
+
+## 4. Deploy the first cluster node. 
+
+Deployment involves installing the operating system and other components on the host, and then rebooting once installation is completed. Deploying the first node creates the cluster, and the first node is assigned the management node by default. 
+    
+During installation, you must configure node settings, define the cluster management address (VIP) and the cluster token, and specify other information. If necessary, you can configure more settings related to installation, the operating system, and advanced features using a [Harvester configuration](../install/harvester-configuration/) file. 
+
+Once installation is completed, the node restarts and then the Harvester console appears. The console displays information about the cluster (management URL and status) and the node (hostname, IP address, and status). After the cluster is initialized and all services start running, the cluster status changes to **Ready**. 
+
+## 5. Configure a strong password for the default `admin` user on the Harvester UI. 
+
+Once the cluster status changes to **Ready**, you can access the [Harvester UI](../authentication) using the management URL displayed on the console. 
+
+## 6. Deploy the other nodes and join them to the cluster. 
+
+Deployment involves installing the operating system and other components on the host, and then rebooting once installation is completed. All other nodes join the cluster that was created when the first node was deployed. 
+
+During installation, you must configure node settings, and specify the cluster management address (virtual IP) and the cluster token that you defined previously. If necessary, you can configure more settings related to installation, the operating system, and advanced features using a [Harvester configuration](../install/harvester-configuration/) file. 
+
+When the cluster has three or more nodes, the two nodes added after the first node are automatically promoted to management nodes to form a high-availability (HA) cluster. 
+
+## 7. Networking? 
```

Comment:
@bk201 Are there any networking-related actions that users must perform before creating VMs?

Reply:
Yes. To make a VM useful, a user should configure a VLAN network first. Our network document ([ref1](https://docs.harvesterhci.io/v1.3/networking/index), [ref2](https://docs.harvesterhci.io/v1.3/networking/harvester-network)) is scattered and we want a small section in the getting start doc to help a new user configure network quickly. The paragraph can include
- A network plan.
- Create a cluster network.
- And create a VM network.

cc @mingshuoqiu @starbops too